packages/ember-metal/lib/alias.js

```diff
@@ -20,66 +20,67 @@ export default function alias(altKey) {
   return new AliasedProperty(altKey);
 }
 
-export function AliasedProperty(altKey) {
-  this.isDescriptor = true;
-  this.altKey = altKey;
-  this._dependentKeys = [altKey];
-}
+export class AliasedProperty extends Descriptor {
+  constructor(altKey) {
+    super();
```

Comment:
@krisselden / @chadhietala do we do the classCheck removal thing on the ember codebase? I don't recall, but you two may.

We should verify the associated compiled output for this is OK before we merge this one.